packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/SelectedEntriesModal/index.js

```diff
@@ -188,6 +188,7 @@ const SelectedEntriesTableContent = ({ isPublishing, rowsToDisplay, entriesToPub
                 )}
                 noBorder
                 target="_blank"
+                style={{ marginLeft: 'auto' }}
```

Comment:
Since `IconButton` inherits from `BaseButton` and `BaseButton` from `Flex` you should be able to use the `marginLeft` prop instead?

Reply:
done